### ISSUE TITLE
Photo feed UI: Users should be able to flag content in the app

### ISSUE THREAD

**Issue by perpetuatingcuriosity** [creativecommons/list#320]
Should work as shown in invision prototype**: http://invis.io/MH3FLVEB7

Right now I’m assuming: Flagging removes the item from a person’s personal feed, but may exist in others’ feeds until a flag notice is reviewed. 

**@sarahpearson: I know we wanted to make it harder to take down potentially misattributed content. For misattribution or copyright flags, is there a URL we can send people to file the full complaint you’d mentioned? Or do we want to leave out this type of flag in our initial release?

**Comment by perpetuatingcuriosity** [comment]
Layouts + UX for basic userflow is in place, i.e.: a user can say: “I don’t want to see this”, a dialog pops up (“are you sure you want to do this?”), and user will confirm or deny. 

Will require more work and testing after #319

**Comment by perpetuatingcuriosity** [comment]
Closing this in favour of #337
